specification/electronic-prescription-service-api.yaml

```diff
@@ -133,8 +133,48 @@ info:
     We are hoping to make the onboarding process as lightweight and as self-service as possible.
 
     More details to follow.
-
-      
+    
+    ## Responses
+    ### 2xx responses
+    **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    | Endpoint              | HTTP code | Response format                                                                                                                                             |
+    | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$convert        | 200       | [See /$convert response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-convertPrescription)      |
+    | POST /$process-message| 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$convert        | 200       | [See /$convert response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-convertPrescription)      |
+    | POST /$process-message| 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
```

Comment:
more a question but the url this points to is a bit rubbish - we renamed the `send` endpoint to `$process_message` so would be good to update anywhere we use send to process_message

Reply:
not sure where this url is actually set?

Reply:
yea me neither lol maybe just keep it in mind?